modules/statics/src/account.ts

```diff
@@ -154,6 +156,7 @@ export function account(
   decimalPlaces: number,
   asset: UnderlyingAsset,
   features: CoinFeature[] = AccountCoin.DEFAULT_FEATURES,
+  primaryKeyCurve: SupportedKeyCurve = SupportedKeyCurve.Secp256k1,
```

Comment:
Why add in the middle of the args list?

Reply:
so if you look at how we instantiate coins in `coins.ts`, I'd have to pass undefined like a ton of times. I don't mind moving it or we'd have to destructure these constructor params. I just think it was out of scope for this PR to change all this scaffolding